tests/components/command_line/test_sensor.py

```diff
@@ -157,29 +147,25 @@ def test_update_with_missing_json_attrs(self):
         )
         self.sensor.update()
         assert "some_json_value" == self.sensor.device_state_attributes["key"]
-        assert (
-            "another_json_value" == self.sensor.device_state_attributes["another_key"]
-        )
-        assert "value_three" == self.sensor.device_state_attributes["key_three"]
+        assert ("another_json_value" ==
+                self.sensor.device_state_attributes["another_key"])
+        assert "value_three" == self.sensor.device_state_attributes[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)